bpf/lib/nodeport.h

```diff
@@ -1548,6 +1572,18 @@ static __always_inline int dsr_reply_icmp4(struct __ctx_buff *ctx,
 	if (ctx_store_bytes(ctx, off + sizeof(ip), &icmp,
 			    sizeof(icmp), 0) < 0)
 		goto drop_err;
+#if DSR_ENCAP_MODE == DSR_ENCAP_NONE
```

Comment:
I'm missing something, why do we want this to happen only for this purpose?

Reply:
No. The change is applicable in any encap mode. Thanks for bringing it up.